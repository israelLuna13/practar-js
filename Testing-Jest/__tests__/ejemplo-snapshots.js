const cliente = {
    nombre:'Juan 2',
    balance:500,
    tipo:'Premium'
}

describe("Testing al cliente",()=>{
    test('Es Israel',()=>{
        expect(cliente).toMatchSnapshot()
    })
})