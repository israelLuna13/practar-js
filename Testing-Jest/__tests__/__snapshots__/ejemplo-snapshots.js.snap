// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Israel 1`] = `
{
  "balance": 500,
  "nombre": "Juan 2",
  "tipo": "Premium",
}
`;
